pyvista/core/filters.py

```diff
@@ -3903,6 +3903,100 @@ def ray_trace(poly_data, origin, end_point, first_point=False, plot=False,
 
         return intersection_points, intersection_cells
 
+
+    def multi_ray_trace(poly_data, origins, directions, first_point=False, retry=False):
+        """Perform multiple ray trace calculations.
+
+        This requires a mesh with only triangular faces,
+         an array of origin points and an equal sized array of
+         direction vectors to trace along.
+
+        The embree library used for vectorisation of the ray traces is known to occasionally
+        return no intersections where the VTK implementation would return an intersection.
+        If the result appears to be missing some intersection points, set retry=True to run a second pass over rays
+        that returned no intersections, using the VTK ray_trace implementation.
+
+
+        Parameters
+        ----------
+        origins : np.ndarray or list
+            Starting point for each trace.
+
+        directions : np.ndarray or list
+            direction vector for each trace.
+
+        first_point : bool, optional
+            Returns intersection of first point only.
+
+        retry : bool, optional
+            Will retry rays that return no intersections using the ray_trace
+
+        Return
+        ------
+        intersection_points : np.ndarray
+            Location of the intersection points.  Empty array if no
+            intersections.
+
+        intersection_rays : np.ndarray
+            Indices of the ray for each intersection point. Empty array if no
+            intersections.
+
+        intersection_cells : np.ndarray
+            Indices of the intersection cells.  Empty array if no
+            intersections.
+
+        Examples
+        --------
+        Compute the intersection between rays from the origin in directions
+        [1, 0, 0], [0, 1, 0] and [0, 0, 1], and a sphere with radius 0.5 centered at the origin
+
+        >>> import pyvista as pv # doctest: +SKIP
+        ... sphere = pv.Sphere()
+        ... points, rays, cells = sphere.multi_ray_trace([[0, 0, 0]]*3, [[1, 0, 0], [0, 1, 0], [0, 0, 1]], first_point=True)
+        ... string = ", ".join([f"({point[0]:.3f}, {point[1]:.3f}, {point[2]:.3f})" for point in points])
+        ... print(f'Rays intersected at {string}')
+        Rays intersected at (0.499, 0.000, 0.000), (0.000, 0.497, 0.000), (0.000, 0.000, 0.500)
+        """
+        if not poly_data.is_all_triangles():
+            raise NotAllTrianglesError
+
+        try:
+            import trimesh, rtree, pyembree
+        except (ModuleNotFoundError, ImportError):
+            raise ImportError(
+                "To use multi_ray_trace please install trimesh, rtree and pyembree with:\n"
+                "\tconda install trimesh rtree pyembree\nor:\n"
+                "\tpip install trimesh rtree pyembree\n\n"
```

Comment:
```suggestion
                "\tconda install trimesh rtree pyembree\nor:\n"
```

There's no pip package for `pyembree`.  Let's just keep it to `conda` for now and we can add this later should they add support for this.